_includes/join-us/join-us-contribute-to-strimzi-band.html

```diff
@@ -0,0 +1,123 @@
+
+      <p>You can also help us to improve the quality of our releases and help us test Strimzi. 
+        As most modern projects, we rely heavily on <a href="https://strimzi.io/blog/2020/09/21/introduction-to-system-tests/">automated tests</a>. 
+        You can always help us improve the tests - you can add new tests to improve the test coverage, improve their execution time and so on. 
+        But you can also help by manually testing Strimzi in different configurations, environments and in different use-cases and report the results and any issues you found.</p>
+
+      <p>Every Strimzi release starts with a release candidate. 
+        Testing the RCs in your environments with your configurations and making sure everything works before the final release is done is also very helpful. 
+        Fixing any bugs before the release is always better and easier than doing patch releases.</p>
+
+      <h3>Write code</h3>
+
+      <p>Since Strimzi is a software project, you can of course also contribute by writing code. 
+        One of the things which makes Strimzi different from most other cloud native projects is that most of its parts are written in Java. 
+        So if you want to contribute to an exciting project and work with modern technologies such as Kubernetes but do not want to learn Golang, then Strimzi is the right project for you. 
+        And if you are more interested in Apache Kafka than in Kubernetes, you will find Strimzi interesting as well.</p>
+
+      <p>Our <a href="https://github.com/strimzi/strimzi-kafka-operator/blob/main/development-docs/DEV_GUIDE.md">developer guide</a> should help you to learn how to develop and build the code. 
+        If you are using Strimzi, you can contribute features or improvements which you are missing. 
+        You can also check the <a href="https://github.com/strimzi/strimzi-kafka-operator/issues">open issues on GitHub</a> and fix some bugs or implement some enhancements.
+        We always try to have some <a href="https://github.com/strimzi/strimzi-kafka-operator/issues?q=is%3Aopen+is%3Aissue+label%3Agood-start">issues labeled as <i>good start</i></a> which are suitable for beginners.
+        If you plan to work on some bigger contribution, it might be good idea to check with us first - for example in the <a href="https://slack.cncf.io/"><code>#strimzi-dev</code> channel on CNCF Slack</a>. 
+        Bigger changes with major impact on the APIs might require <a href="https://github.com/strimzi/proposals">writing a proposal</a> and having it approved by the community. 
+        For small changes and bug fixes, you can just open the PR and we will review it. 
+        Programming also doesn’t always mean just new features or bugfixes. 
+        You can also contribute to our tests for example by writing <a href="https://strimzi.io/blog/2020/12/03/how-the-system-tests-works/">new system tests</a>.</p>
```

Comment:
```suggestion
        If you plan to work on a bigger contribution, it might be good idea to check with us first. You could drop your suggestion in the <a href="https://slack.cncf.io/"><code>#strimzi-dev</code> channel on CNCF Slack</a>. 
        Bigger changes with major impact on the APIs might require <a href="https://github.com/strimzi/proposals"> a proposal</a> that's approved by the community. 
        For small changes and bug fixes, you can just open the PR and we will review it. 
        Programming also doesn’t always mean just new features or bugfixes. 
        You can also contribute to our tests by writing <a href="https://strimzi.io/blog/2020/12/03/how-the-system-tests-works/">new system tests</a>.</p>
```